sources/packages/backend/apps/api/src/route-controllers/application-change-request/_tests_/e2e/application-change-request.aest.assessApplicationChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,482 @@
+        },
+      }),
+    );
+    // Validate database changes.
+    const applicationChangeRequest = await db.application.findOne({
+      select: {
+        id: true,
+        applicationStatus: true,
+        applicationStatusUpdatedOn: true,
+        currentAssessment: {
+          id: true,
+          offering: {
+            id: true,
+          },
+          studentAppeal: {
+            id: true,
+          },
+        },
+        modifier: {
+          id: true,
+        },
+        updatedAt: true,
+        precedingApplication: {
+          id: true,
+          applicationStatus: true,
+          applicationStatusUpdatedOn: true,
+          applicationEditStatus: true,
+          applicationEditStatusUpdatedOn: true,
+          currentAssessment: {
+            id: true,
+            offering: {
+              id: true,
+            },
+            studentAppeal: {
+              id: true,
+            },
+          },
+          modifier: {
+            id: true,
+          },
+          updatedAt: true,
+        },
+      },
+      relations: {
+        currentAssessment: {
+          offering: true,
+          studentAppeal: true,
+        },
+        modifier: true,
+        precedingApplication: {
+          currentAssessment: {
+            offering: true,
+            studentAppeal: true,
+          },
+          modifier: true,
+        },
+      },
+      where: {
+        id: changeRequest.id,
+      },
+      loadEagerRelations: false,
+    });
+    expect(applicationChangeRequest).toEqual({
+      id: changeRequest.id,
+      applicationStatus: ApplicationStatus.Completed,
+      applicationStatusUpdatedOn: now,
+      currentAssessment: {
+        id: expect.any(Number),
+        offering: expectedOffering,
+        studentAppeal: expectedAppeal,
+      },
+      modifier: ministryUser,
+      updatedAt: now,
+      precedingApplication: {
+        id: applicationToBeReplaced.id,
+        applicationStatus: ApplicationStatus.Edited,
+        applicationStatusUpdatedOn: now,
+        // Ensure value was not updated.
+        applicationEditStatus: applicationToBeReplaced.applicationEditStatus,
+        // Ensure value was not updated.
+        applicationEditStatusUpdatedOn:
+          applicationToBeReplaced.applicationEditStatusUpdatedOn,
+        currentAssessment: {
+          id: applicationToBeReplaced.currentAssessment.id,
+          offering: expectedOffering,
+          studentAppeal: expectedAppeal,
+        },
+        modifier: ministryUser,
+        updatedAt: now,
+      },
+    });
+    // Validate created student note.
+    const student = await db.student.findOne({
+      select: {
+        id: true,
+        notes: { id: true, description: true, creator: { id: true } },
+      },
+      relations: {
+        notes: { creator: true },
+      },
+      where: {
+        id: changeRequest.student.id,
+      },
+      loadEagerRelations: false,
```

Comment:
Do we need this?

Reply:
Yes, otherwise, the user data will be loaded because its model is configured to `eager` load.
There are other E2Es in the source code doing the same when we need to validate the user.